tests/screenshot/apps/components/MediaPlayer.js

```diff
@@ -4,6 +4,9 @@ const MediaPlayerTests = [
 	<MediaPlayer>
 		<source src="https://sampleswap.org/mp3/artist/254731/BossPlayer_Your-Right-Here-160.mp3" type="audio/mp3" />
 	</MediaPlayer>,
+	<MediaPlayer spotlightDisabled>
```

Comment:
```suggestion
	<MediaPlayer disabled>
```

The spotlightDisabled property doesn't change any looks, so I don't think we can verify via screenshot test. The ui test is more appropriate.